pkg/runner/run_context.go

```diff
@@ -684,13 +715,8 @@ func (rc *RunContext) isEnabled(ctx context.Context) (bool, error) {
 
 	img := rc.platformImage(ctx)
 	if img == "" {
-		if job.RunsOn() == nil {
-			l.Errorf("'runs-on' key not defined in %s", rc.String())
-		}
-
-		for _, runnerLabel := range job.RunsOn() {
-			platformName := rc.ExprEval.Interpolate(ctx, runnerLabel)
-			l.Infof("\U0001F6A7  Skipping unsupported platform -- Try running with `-P %+v=...`", platformName)
+		for _, platformName := range rc.runsOnPlatformNames(ctx) {
+			l.Infof("\U0001F6A7  Skipping unsupported platform -- Try running with `-P %+v=...`", strings.ToLower(platformName))
```

Comment:
added `ToLower` here to make the error messaging consistent with the existing (but I think undocumented?) behavior, i.e. any -P option needs to be lower case, since we only ever look up lower-cased keys in `rc.Config.Platforms`

Reply:
Went ahead and just fixed the case sensitivity bug here since it was trivial... we'll see if the maintainer wants that in a separate change though